src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -162,6 +163,7 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine($"#agentprefix=<string>");
             builder.AppendLine($"#Enable bandwidth saving setting to send and received confirmed blocks only. Defaults to { (DefaultBlocksOnly ? 1 : 0) }.");
             builder.AppendLine($"#blocksonly={ (DefaultBlocksOnly ? 1 : 0) }");
+            builder.AppendLine($"#bantime=<number>");
```

Comment:
It seems like every line here has an explanation line before it, so we should have an explanation line before this one as well

Reply:
As above ^^^